.github/ISSUE_TEMPLATE/2-issue-op-cli.yml

```diff
@@ -0,0 +1,31 @@
+name: "💲 1Password CLI overall shell plugins functionality"
+description: I have an issue or feature request around using shell plugins in 1Password CLI.
+labels: ["op-cli"]
+body:
+  - type: input
+    id: version
+    attributes:
+      label: op CLI version
+      description: "Add the output of: op --version"
+    validations:
+      required: true
+  - type: textarea
+    id: desired
+    attributes:
+      label: Goal or desired behavior
+      description: What are you trying to achieve in 1Password CLI that's currently not working well?
```

Comment:
Since this repository is for 1Password Shell Plugins, should it be In `1Password CLI` or `with a shell plugin`?